src/App.test.tsx

```diff
@@ -50,6 +53,20 @@ describe('App', () => {
     expect(map).toBeInTheDocument();
   });
 
+  it('should redirect to /map if no data returned for city name', async () => {
```

Comment:
should it be a part of `App.test.tsx` or `CityPage.test.tsx` 🤔

Reply:
I think it's better to move it to `CityPage.test.tsx`, thanks